vars/buildPlugin.groovy

```diff
@@ -48,224 +48,227 @@ def call(Map params = [:]) {
     }
 
     tasks[stageIdentifier] = {
-      node(label) {
```

Comment:
You need to ignore WS diffs in this PR. (I tried to just use one-space indentation but Spotless shot me down.)